sky/data/storage.py

```diff
@@ -1689,7 +1704,15 @@ def _validate(self):
                 assert self.name == data_utils.split_r2_path(self.source)[0], (
                     'R2 Bucket is specified as path, the name should be '
                     'the same as R2 bucket.')
+        # Validate name
         self.name = S3Store.validate_name(self.name)
+        # Check if the storage is enabled
+        enabled_storage_clouds = global_user_state.get_enabled_storage_clouds()
+        if 'Cloudflare' not in enabled_storage_clouds:
```

Comment:
Is there some better solution than having a hardcoded string `Cloudflare` here? Ideally cloudflare would have been a cloud and we could have called `str(clouds.Cloudflare())` (Related to #1943, sigh). 

Can we have this `'Cloudflare'` as a str in `cloudflare.py`, something like `NAME = 'Cloudflare'`? Then we can use it here and elsewhere as `Cloudflare.NAME`. Also open to other suggestions.

Reply:
Since `adaptors/cloudflare.py` currently contains the temporary solution code that could be in `clouds.Cloudflare`, I think it makes sense to have `NAME` in this file that can be refactored later.